src/constants.js

```diff
@@ -254,6 +254,7 @@ export const defaultTheme = {
       "40<": darkGreen6,
       [DEMOGRAPHIC_UNKNOWN]: darkGreen5,
     },
+    asideBackground: "rgba(206, 202, 199, 0.8)", // "#CECAC7"
```

Comment:
I needed the actual alpha channel to be used here in order to allow for some transparency in the aside overlay.